alacritty_terminal/src/term/cell.rs

```diff
@@ -30,6 +30,7 @@ bitflags! {
         const ALL_UNDERLINES            = Self::UNDERLINE.bits | Self::DOUBLE_UNDERLINE.bits
                                         | Self::UNDERCURL.bits | Self::DOTTED_UNDERLINE.bits
                                         | Self::DASHED_UNDERLINE.bits;
+        const PROMPT_MARK               = 0b1000_0000_0000_0000;
```

Comment:
Should be above `ALL_UNDERLINES`. The "special" constants are usually best kept at the bottom because that way one can easily see at a glance that all bits are correctly set for the other constants.